internal/domain/domain.go

```diff
@@ -17,7 +17,7 @@ func MainDomain(ctx context.Context) (string, error) {
 		return mainDomain, nil
 	}
 
-	cluster, err := kubernetes.GetCluster(ctx)
+	cluster, err := kubernetes.GetCluster(ctx) // Push -> MainDomain -> here (client-side! FIXIT?)
```

Comment:
fixed already?

Reply:
Yes. Should remove that.

Reply:
Addressed by commit a867f2312964f497cb812b873512d16b475c3c4a